server/planning/archive.py

```diff
@@ -38,6 +38,12 @@ def create_item_from_template(doc, extra_fields_to_override=None):
     # and apply them if any found
     updates = {key: val for key, val in doc.items() if key in fields_to_override}
 
+    # override translated values if any
+    translated_value = {
+        entry["field"]: entry["value"] for entry in translations or [] if entry["language"] == doc.get("language")
+    }
+    updates = {key: val for key, val in translated_value.items() if key in fields_to_override}
```

Comment:
This is overwriting not extending what the original `updates` value is. This is probably why you're getting failing tests.